DEVELOPMENT.md

```diff
@@ -1,51 +1,51 @@
- - Not all committers will contribute to every single proposed change.  There may be many open proposals at once, and multiple efforts may happen in parallel
+ - The `master` branch shall remain the current, released OpenAPI Specification (i.e., 2.0).  We will work in an OpenAPI.next branch, which shall be described and linked to on the **default** README.md on master.
+ - Examples of how something is described _currently_ vs. the proposed solution should accompany any change proposal.
+ - New features should be done in feature branches which, upon approval, are merged into the OpenAPI.next branch.
+ - Use labels for the workflow of specification changes.  Examples of labels are `proposed`, `needs migration review`, `needs tooling review`, `needs documentation`, `rejected`, and `needs approval`.  These labels must be assigned by project committers.
+ - An issue will be opened for each feature change.  Embedded in the issue, or ideally linked in a file via pull-request (PR), a document about use cases should be supplied with the change.
+ - A PR will be used to describe the _proposed_ solution, and linked to the original issue.
+ - Not all committers will contribute to every single proposed change.  There may be many open proposals at once, and multiple efforts may happen in parallel.
  - When the OpenApi.next spec is complete and approved for release, the branch will be merged to master.
 
 ## Approving Changes
 
 For each change in the specification we should _always_ consider the following:
 
  - Migration.  Is this a construct that has a path from the existing 2.0 specification?  If so, how complicated is it to migrate to the proposed change?
- - Tooling.  Strive to support code generation, software interfaces, spec generation techniques.  Some features may be impossible to support in different frameworks/languages.  These should be documented and considered if the change should be approved.
- - Visualization.  Can the specification change be graphically visualized somehow in a UI or other?
+ - Tooling.  Strive to support code generation, software interfaces, and spec generation techniques.  Some features may be impossible to support in different frameworks/languages.  These should be documented and considered during the change approval process.
+ - Visualization.  Can the specification change be graphically visualized somehow in a UI or other interface?
 
-Spec changes should be approved by a majority of the committers.  This can be done by commenting on the issue itself ("Approved by @fehguy" for example).  Once voting criteria is met, any committer can merge the PR. (**TODO: we will want to formalize what voting criteria actually is).
+Spec changes should be approved by a majority of the committers.  Approval can be given by commenting on the issue itself, for example, "Approved by @fehguy".  After voting criteria is met, any committer can merge the PR. (**TODO: we will want to formalize what voting criteria actually is).
```

Comment:
If you are changing this paragraph, could you close the `**` here after the TODO? I guess markdown's in-line markup should close itself at the end of the paragraph, but Github's diff view doesn't seem to know this.)